proto/protos/remote_engine.proto

```diff
@@ -24,6 +24,7 @@ message TableIdentifier {
 message ReadOptions {
   uint64 batch_size = 1;
   uint64 read_parallelism = 2;
+  common.U64Wrapper timeout_ms = 3;
```

Comment:
Maybe it is more elegant to use int64 as the type `timeout_ms` so that we can use a negative timeout to express no deadline.

Reply:
Fixed